packages/oui-message/README.md

```diff
@@ -220,6 +220,22 @@ oui-message is a package which provide styles for the `oui-message` component.
 </div>
 ```
 
+### Large
+
+```html:preview
```

Comment:
But the documentation would be broken, we need it to generate a preview. It is the only way we found to not duplicate the html.